.travis.yml

```diff
@@ -197,6 +197,8 @@ matrix:
         - docker
       env: JOB_TYPE=external_prerelease
       before_script:
+        - sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros.list'
```

Comment:
Are these travis changes still required for all these tests after moving the import into the ABI checker-specific method?

Reply:
you are totally right, I think they are not needed anymore, removed here bac1c75